main.py

```diff
@@ -1,12 +1,210 @@
+
+        # Option 6: Create a new customer 
+        elif choice == '6':
+            print("Great! Let's create a new customer.")
+            name = input("Name of the customer: ")
+            postal_code = input("Customer's postal_code: ")
+            phone = input("Customer's phone number: ")
+
+            response = video_store.create_customer(name=name, postal_code=postal_code, phone=phone) 
+            print("New customer: #{response['id']} has been added.") 
+        
+        # Option 7: Edit a customer 
+        elif choice == '7':
+            print("Great! Let's update the customer.")
+            name = input("New name of the customer: ")
+            postal_code = input("Customer's new postal_code: ")
+            phone = input("Customer's new phone number: ")
+
+            response = video_store.update_customer(name=name, postal_code=postal_code, phone=phone) 
+            print("Updated customer: #{response['id']} has been updated.") 
+
+        # Option 8: Delete a customer 
+        elif choice =='8':
+            delete_customer_id = input("Enter the customer id to delete: ")
+            if delete_customer_id.isnumeric():
+                video_store.delete_customer(id=delete_customer_id)
+            print("Customer id #{delete_customer_d} has been deleted.")
+            
+        
+        # Option 9: Get information of all customers 
+        elif choice =='9':
+            print("Customer: ")
+            print_stars()
+            for customer in video_store.list_customer():
+                print(customer)
+
+        # Option 10: Get information of one customer 
+        elif choice =='10':
+            select_by = input("Select by customer ID: ")
+            if select_by.isnumeric():
+                customer = video_store.get_customer(id=customer_id)
+                print(customer)
+
+
+# ==========Video and Customer: Checkin/Check-out========== 
+
+        # Option 11: Check out a video to a customer 
+        elif choice=='11':
+            print("You are checking out a video!")
+            video_id = input("Video id: ")
+            customer_id = input("Customer id: ")
+
+            if customer_id == None or video_id == None:
+                print("No record found.")
+            else: 
+                checkout = video_store.check_out(customer_id=customer_id, video_id=video_id)
+            print("You checked out #{video_id} to customer #{customer_id}")
+    
+        # Option 12: Check in video from a customer 
+        elif choice=='12':
+            print("You are checking in a video!")
+            video_id = input("Video id: ")
+            customer_id = input("Customer id: ")
+
+            if customer_id == None or video_id == None:
+                print("No record found.")
+            else: 
+                checkin = video_store.check_in(customer_id=customer_id, video_id=video_id)
+            print("You checked in video #{video_id} from a customer #{customer_id}.")
```

Comment:
print statements like this one aren't actually printing out the ids. `f""` syntax needs to be used in Python to use string interpolation.